sdk/loadtestservice/loadtestservice/test/public/appComponents.spec.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Recorder } from "@azure-tools/test-recorder";
+import { assert } from "chai";
+import { createRecorder, createClient } from "./utils/recordedClient";
+import { Context } from "mocha";
+import { AzureLoadTestingClient } from "../../src";
+
+describe("File Upload", () => {
+  let recorder: Recorder;
+  let client: AzureLoadTestingClient;
+  const SUBSCRIPTION_ID = process.env["SUBSCRIPTION_ID"] || "";
+
+  beforeEach(async function (this: Context) {
+    recorder = await createRecorder(this);
+    client = createClient();
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it("should upload the test file", async () => {
+    const result = await client.path("/appcomponents/{name}", "appcomp123").patch({
```

Comment:
Just out of curious, why use patch to create the resource ?

Reply:
We are not doing any azure resource control plane operation in this, it is our internal entity called app component, which is similar to a Azure resource. We provide a functionality where customer can see the resource metrics, so to pull the azure monitor metrics we need azure resource information. So to store these information customer can do patch on app component entity to link their azure resources with our load test run.